package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hpagent",
-  "version": "0.1.2",
+  "version": "0.1.3",
```

Comment:
Please avoid updating the version number in a pr, it will be done separately.

Reply:
Gotcha, will do

Reply:
> Great work! Can you move the certificates in `test/fixtures` and then add `test/fixtures` to `.gitignore?`

For the certificates, if they are gitignored, then the CI test would fail right? Since the certs need to be loaded when the proxy / http server are "created" in the test environment

Reply:
(similar to the previous files `ssl.key` and `ssl.cert`)

Reply:
You are right! You should create a copy of the `.gitignore` file and call it `.npmignore` and add to it the `test/fixtures` folder :)

Reply:
Cool, lets see if it works!